core/src/main/scala/org/bitcoins/core/util/Bech32.scala

```diff
@@ -7,6 +7,21 @@ import scodec.bits.ByteVector
 import scala.annotation.tailrec
 import scala.util.{Failure, Success, Try}
 
+sealed abstract class Bech32Encoding {
```

Comment:
add a link to what/where these differences are specified. I'm assuming they are on in the BIP?